### ISSUE TITLE
Webhooks with JWT - Authorization header using the Bearer schema

### ISSUE THREAD

**Issue by paul-brenner** [ZekeSnider/Jared#85]
I'd love to be able to add a header with Authorization: Bearer xxxxxxxx to make things a little more secure. It seems like that is something that would make sense to set in the config then pass in, I'm guessing [here](https://github.com/ZekeSnider/Jared/blob/969f145321deacfc41401fb7728053f240dad465/Jared/WebHookManager.swift#L55). 

I assume since no one else has asked about this there isn't really any demand for it and I should just hack it together myself, but figured I would ask.

**Comment by ZekeSnider** [comment]
Yep that spot looks right to me. For a clean solution we'd probably want an optional Bearer config field for each configured webhook. PRs welcome if you'd like to, hacking it together would work otherwise.
